x-pack/plugins/canvas/public/state/defaults.js

```diff
@@ -29,7 +29,7 @@ export const getDefaultPage = () => {
   return {
     id: getId('page'),
     style: {
-      background: '#fff',
+      background: '',
```

Comment:
I get why this was done, to make it possible to override the background color of every page in the workbook.

However, if the user ever selects a color from the page style chooser they won't be able to unset it without hacking the JSON. What about just adding a "Apply to All" button next to the page background color chooser? Then the user wouldn't need workpad level CSS to set a global background.

Reply:
After chatting with @cqliu1 , I've changed this back to white as the default. She informed me that she already has plans to tackle 'Apply to all' in a separate issue: https://github.com/elastic/kibana/issues/23933